network-logs/minio-server/data/.minio.sys/format.json

```diff
@@ -0,0 +1 @@
+{"version":"1","format":"fs","id":"b43d434a-ae8b-4ed4-a7a6-0d80eda86ee8","fs":{"version":"2"}}
```

Comment:
Why do we need to check in a logs or data folder? This seems like an anti-pattern and makes it easy to check in files we should not that might contain sensitive information. It should just be created by whichever component that needs it and gitignored.

Reply:
The `minio-server/data` folder is added to create `hedera-streams` bucket when the server starts.  When tried using `mb` to create bucket, faced few issues. 
Do you think it should not be done  this way ? 

Agreed, will add this folder to gitignore

Reply:
We should auto-create the bucket, but the network-logs folder doesn't need to be checked in to do this. I think minio will auto create buckets for any folder under `/data`. So you just need to modify the entrypoint/command to create the dir if it doesn't exist. There's a number of solutions [here](https://github.com/minio/minio/issues/4769). Something like this:

```yaml
    entrypoint: sh
    command: -c 'mkdir -p /data/hedera-streams && /usr/bin/minio server /data'
```

Reply:
Fixed as Steven suggested using the command 
`command: -c 'mkdir -p /data/hedera-streams && minio server /data --console-address ":9001"'`